modules/ROOT/pages/dynamic-expression-field.adoc

```diff
@@ -75,7 +75,7 @@ The Logger will add a message in this format to the log when it receives a paylo
 "Hello, Ian."
 ----
 +
-If the payload contains a `lastName` data pill after the `firstName` data pill and a space is added between them, the Logger will log messages in this format:
+If the payload contains a `lastName` data element after the `firstName` data element and a space is added between them, the Logger will log messages in this format:
```

Comment:
Changer "will log" to "log"

Reply:
Then, "If the payload..., the Logger **log** messages in this format:"? Thanks for the suggestion, but I disagree with it.